includes/dotnet-restore-note.md

```diff
@@ -1 +1,2 @@
-**Note:** Starting with .NET Core 2.0, you do not have to run [`dotnet restore`](~/docs/core/tools/dotnet-restore.md) because it is run implicitly as part of `dotnet build` or `dotnet run`. It is still a valid command in certain scenarios where doing an explicit restore makes sense, such as [continuous integration builds in Visual Studio Team Services](https://www.visualstudio.com/team-services/support/build-deploy-asp-net-core-app-azure/).
+> [!NOTE]
```

Comment:
@mairaw, the link is broken in the build.

Reply:
@rpetrusha hopefully it will work now.